pyfr/solvers/euler/kernels/rsolvers/rsolve1d.mako

```diff
@@ -0,0 +1,21 @@
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
```

Comment:
I'd ditch this and just inline the solver since it is the only one of its kind.

Reply:
As above, this framework makes many of the other Riemann solvers easier to implement. This is especially true for other systems, so I think it should stay here for utility.

Reply:
Okay.  I think the best approach is to port one of them over, and then make that a PR.  The exact Riemann solver can then bolt on later as a separate PR (as it is somewhat more complicated).